templates/create_ag.j2

```diff
@@ -0,0 +1,43 @@
+-- Need to find how to alter ag idempotently. FOr now - drop the ag and re-create if exists
+IF EXISTS (SELECT * FROM sys.availability_groups WHERE name = '{{ mssql_ha_ag_name }}')
+BEGIN
+  PRINT 'Availability group {{ mssql_ha_ag_name }} already exists, removing';
+  DROP AVAILABILITY GROUP {{ mssql_ha_ag_name }};
+  PRINT 'Availability group {{ mssql_ha_ag_name }} removed successfully';
+END
+
+PRINT 'Creating the {{ mssql_ha_ag_name }} availability group';
+CREATE AVAILABILITY GROUP {{ mssql_ha_ag_name }}
+{% if ansible_os_family == 'RedHat' and ansible_distribution_version | float < 8.3 %}
```

Comment:
Use the `version` test instead of `float` - https://docs.ansible.com/ansible/2.9/user_guide/playbooks_tests.html#version-comparison

Reply:
Done, didn't know this thank you